index.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<!--
+Copyright 2018 The Go Authors. All rights reserved.
+Use of this source code is governed by a BSD-style
+license that can be found in the LICENSE file.
+-->
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<title>Go wasm</title>
+</head>
+
+<body>
+	<!--
+	Add the following polyfill for Microsoft Edge 17/18 support:
+	<script src="https://cdn.jsdelivr.net/npm/text-encoding@0.7.0/lib/encoding.min.js"></script>
+	(see https://caniuse.com/#feat=textencoder)
+	-->
+	<script src="wasm_exec.js"></script>
+	<script>
+		if (!WebAssembly.instantiateStreaming) { // polyfill
+			WebAssembly.instantiateStreaming = async (resp, importObject) => {
+				const source = await (await resp).arrayBuffer();
+				return await WebAssembly.instantiate(source, importObject);
+			};
+		}
+
+		let exitCode = 0;
+		function goExit(code) {
+			exitCode = code;
+		}
+		function enosys() {
+			const err = new Error("not implemented");
+			err.code = "ENOSYS";
+			return err;
+		}
+		let coverageProfileContents;
+
+		(async() => {
+			const go = new Go();
+			const defaultWriteSync = window.fs.writeSync;
+			const coverFileDescriptor = 3; // 1 above standard descriptors (0-2)
+			const coverFilePath = {{.CoverageFile}};
+			window.fs.open = (path, flags, mode, callback) => {
+				if (path !== coverFilePath) {
```

Comment:
It's better to invert this logic and write our custom code inside `path === coverFilePath` case, and as advised in the PR thread, it might be safer to retain the original references for open and close as well, and call them when our condition doesn't match. Keeps everything similar to `writeSync`.

Reply:
Fixed in [ffd220a](https://github.com/agnivade/wasmbrowsertest/pull/41/commits/ffd220a589db2de8a13b45d3c125a01d5446b2be)